### ISSUE TITLE
Drawing with Windows Ink may require a double tap

### ISSUE THREAD

**Issue by Eevoo** [mbrlabs/Lorien#158]
## OS/device including version:
Windows 10 21H2
![image](https://user-images.githubusercontent.com/49214133/174172069-7b8df51c-e27c-4237-85a0-745e9407b4fe.png)

## Issue description:
When using an active stylus using an "HID Compliant Pen" (which I can't remember whether it's a Wacom AES or a MPP touchscreen but either way it uses an active stylus that has pressure sensitivity, an erase button etc), Lorien does not draw strokes when the pen is pressed against the screen. 

Lorien does detect that there is pressure however, as shown in the demo pic below.

Holding the middle, left, or eraser buttons while drawing does not appear to have any effect.

## Repro steps:
1. Attempt to draw with the pen as normal
2. Observe no lines are drawn, but note that there is pressure reported from the program

## Demo graphic:
The strokes on the left are attempted with a single tap and hold to draw, the strokes on the right are attempted with a double-tap and hold to draw.

![GIF 2022-06-16 5-50-56 PM](https://user-images.githubusercontent.com/49214133/174175798-02a9b9a2-4a08-4176-9bdc-815999d2de41.gif)

## Workaround:
Attempt a double-tap to cause Windows to send a mousedown event, then hold the pen down for the stroke.

**Comment by hansemro** [comment]
@Eevoo This could be an issue with how godot engine processes inputs for your stylus. Can you provide additional hardware details? I was unable to replicate this.

**Comment by Eevoo** [comment]
@hansemro sure. It is an HP ProBook 435 x360. https://www.notebookcheck.net/HP-ProBook-x360-435-G7-laptop-review-AMD-Ryzen-also-shines-in-the-business-convertible.494446.0.html and the stylus is https://www.hp.com/us-en/shop/pdp/hp-rechargeable-active-pen-g3#techSpecs which as best I can tell from the Q and A, is Wacom AES based technology.

Once I'm not running around on mobile I can try to pull specific drivers if that helps? Or if there is any other way I could gather mouse down events in a way that would be useful for Godot event handlers I'm happy to that too.

**Comment by lieuzhenghong** [comment]
I have the same problem with a Surface Pen 2.

**Comment by hansemro** [comment]
Can you try both wintab and winink driver options in Lorien? I'd like to know if both have the same issue.

1) Create a shortcut to Lorien.exe
2) Right click, then go to Properties
3) Under Target, append ` --tablet-driver wintab` or ` --tablet-driver winink`
4) Launch from shortcut

![image](https://user-images.githubusercontent.com/40348686/177639322-1ca55489-ea79-41b4-90f0-c532796fc1b2.png)

**Comment by lieuzhenghong** [comment]
@hansemro @Eevoo following up here.

I can confirm @Eevoo's observations exactly:

- `--tablet-driver wintab` does not open a window
- `--tablet-driver winink` opens but does not fix the problem

**Comment by Eevoo** [comment]
Hello all, I tried this same test on the same device with the same stylus, running Lorien 0.6.0. Success! I can now draw with a single tap and hold as expected when I opened this last year. The pressure shown is 0, then 1, and there is no pressure sensitivity at the moment, but for now, I'm happy that there is forward progress on this and can open another incident if I can't play around with the pressure sensitivity settings later and decide I 'must' have that feature. 

Thanks for making such a cool product and sharing it with us web freeloaders!
